lte/gateway/python/integ_tests/federated_tests/fabfile.py

```diff
@@ -199,8 +199,8 @@ def build_feg():
     vagrant_setup('magma', destroy_vm=False)
 
     with cd(feg_docker_integ_test_path_vagrant):
-        run('docker-compose down')
-        run('docker-compose build')
+        run('docker compose down')
```

Comment:
Why is `--compatibility` sometimes needed for `up` `down` but sometimes not?

Reply:
I am not sure what is best. Is the idea that we keep `--compatibility` until v3?